.github/workflows/build.yaml

```
@@ -30,3 +30,10 @@ jobs:
           fetch-depth: 1
       - name: Build
         run: make
+      - name: Run markdownlint with auto-fix
+        uses: DavidAnson/markdownlint-cli2-action@05f32210e84442804257b2a6f20b273450ec8265
```

Note that I left the action version in a comment in my suggestion:

```suggestion
        uses: DavidAnson/markdownlint-cli2-action@05f32210e84442804257b2a6f20b273450ec8265 # v19
```

When dependabot creates a PR to update the SHA, it will update the comment with the version too. I don't know if it will add a comment, if there isn't one, so I think it will be safer to add it right away.

---

See for example https://github.com/lima-vm/lima/pull/3291/files

---

got it , did'nt know that dependabot updates it as well